Recommend installing py3.11 instead of 3.10 (#75)
sicara/sicarator: README.md

  ```
*If you have not previously installed any Python version, you may need to install one and set it as your default Python version before installing Poetry:*
  ```bash
  pyenv update  # fetch the latest list of Python versions

>   pyenv install 3.11  # install the latest Python 3.11
>   pyenv global 3.11  # set this as the default version for your shell (and ignore the system Python)

Reviewer: Cool to add some explanation! But your last comment isn't exact: it's not only for the current shell.
```suggestion
  pyenv install 3.11  # install the latest Python 3.11.x
  pyenv global 3.11  # set your new global default Python version
```
